types/ejson/index.d.ts

```diff
@@ -17,7 +17,7 @@ export function parse(str: string): any;
 export function stringify(obj: any, options?: StringifyOptions): string;
 
 export function toJSONValue(obj: any): string;
-export function fromJSONValue(obj: string): any;
+export function fromJSONValue(obj: Object): any;
```

Comment:
```suggestion
export function fromJSONValue(obj: object): any;
```
https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#number-string-boolean-symbol-and-object